### PR TITLE
Add a new cheat Sheet for 'UC Mascots'

### PR DESCRIPTION
**Pull request by jeremiahkonrad**
Hi,

I'm part of the LA DuckDuckGo Meetup. Please consider this PR and let me know if I need to make any changes--thanks!

------
IA Page: https://duck.co/ia/view/uc_mascots_cheat_sheet

### CODE REVIEW

**Comment by MrChrisW** [comment]
Hi @jeremiahkonrad, thanks for your contribution. Do you mind filling out the [Goodie Pull Request Template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_goodie.md) and adding it to your initial comment. This will provide the community a quick way to review your CheatSheet Goodie. Thanks :+1:

File: share/goodie/cheat_sheets/uc-mascots.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+    "id": "uc_mascots",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/uc-mascots.json]: Please change this to `uc_mascots_cheat_sheet` -- to match the soon-to-come IA Page: 06251508183135

**Comment by jeremiahkonrad** [comment]
Hi @MrChrisW and @moollaza , just letting you know I saw your notes and I'll update today. Thanks!

**Comment by moollaza** [comment]
@jeremiahkonrad ping! Just wanted to remind you that whenever you have a chance, please update this one! 

There's no rush, but we'd love to see it go live!

**Comment by moollaza** [comment]
@jeremiahkonrad Ping! Just wanted to check in again. Stuck on anything?

**Comment by moollaza** [comment]
@jeremiahkonrad Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

For now I'm closing this PR. We will gladly open it up again if you'd like to complete it though.

**Pull request closed without merging**